framework/db/BaseActiveRecord.php

```diff
@@ -311,8 +311,7 @@ public function __get($name)
     public function __set($name, $value)
     {
         if ($this->hasAttribute($name)) {
-            if (
-                !empty($this->_relationsDependencies[$name])
+            if (!empty($this->_relationsDependencies[$name])
```

Comment:
Which rule is responsible for such change? It looks worse and it is inconsistent with similar constructs (like list of arguments in function call).

Reply:
Ideed. i will revert and check the squizz rules responsible.

Reply:
This was it:
```
FILE: /Volumes/Code/yii2/framework/db/BaseActiveRecord.php
-------------------------------------------------------------------------------------------------------------------------------------------------
FOUND 2 ERRORS AFFECTING 2 LINES
-------------------------------------------------------------------------------------------------------------------------------------------------
 314 | ERROR | [x] Expected 0 spaces after opening bracket; newline found (PSR2.ControlStructures.ControlStructureSpacing.SpacingAfterOpenBrace)
 524 | ERROR | [x] Expected 0 spaces after opening bracket; newline found (PSR2.ControlStructures.ControlStructureSpacing.SpacingAfterOpenBrace)
-------------------------------------------------------------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 2 MARKED SNIFF VIOLATIONS AUTOMATICALLY
-------------------------------------------------------------------------------------------------------------------------------------------------
```
The good thing is, with PSR12, there is no change, I wanted to do it in separate commits...